### PR TITLE
Always use latest version from GitHub releases

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

- Update the installation script to always use the latest version for GitHub

## Test Plan

- Manually tested

## Related PRs and Issues

No

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

File: files/var/lib/cloud/scripts/per-instance/01-run-appwrite.sh [hunk 1]
```diff
@@ -2,9 +2,14 @@
...
+echo "=       Installing appwrite $appversion       ="
+echo "==============================================="
+
 
 docker run --rm \
     --volume /var/run/docker.sock:/var/run/docker.sock \
     --volume "$(pwd)"/appwrite:/usr/src/code/appwrite:rw \
-    appwrite/appwrite:1.3 sh -c "install --httpPort=80 --httpsPort=443 --interactive=N"
\ No newline at end of file
+    appwrite/appwrite:$appversion sh -c "install --httpPort=80 --httpsPort=443 --interactive=N"
```
Comment (stnguyen90) [inline files/var/lib/cloud/scripts/per-instance/01-run-appwrite.sh:15]: If we always want to get the latest, we could just use `appwrite/appwrite:latest`, right?
```suggestion
    appwrite/appwrite:latest sh -c "install --httpPort=80 --httpsPort=443 --interactive=N"
```

Comment (lohanidamodar) [reply files/var/lib/cloud/scripts/per-instance/01-run-appwrite.sh:15]: I think that will also create the `docker-compose.yml` file with `apwrite:latest` (I'm not sure) so if docker-compose has latest, if they always run `docker pull latest` there might be confusion as to which version they are at, which version they will be upgrading to, do they need to migrate and all?

Comment (stnguyen90) [reply files/var/lib/cloud/scripts/per-instance/01-run-appwrite.sh:15]: That version value is pulled from an env var in the docker image so it should be okay: 

image

**Review by stnguyen90 — COMMENTED** [review]
Replied in https://github.com/appwrite/integration-for-digitalocean/pull/37/files#r1269704118

File: files/var/lib/cloud/scripts/per-instance/01-run-appwrite.sh (hunk 1 above)
Comment (lohanidamodar) [reply files/var/lib/cloud/scripts/per-instance/01-run-appwrite.sh:15]: Still want to keep this, so even if there is error with updating latest image, we should be good.

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**